control-plane/tests/test_service_persistence.py

```
@@ -9787,6 +10436,200 @@ def test_service_phase20_first_live_action_fail_closes_on_downstream_execution_s
             (execution.action_execution_id,),
         )
 
+    def test_service_reconciliation_rolls_back_execution_state_when_reconciliation_write_fails(
+        self,
+    ) -> None:
+        base_store, seed_service, promoted_case, evidence_id, reviewed_at = (
```

Ruff (0.15.9)

[warning] Unpacked variable `reviewed_at` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)